mesatee_services/kms/proto/Cargo.toml

```diff
@@ -12,6 +12,9 @@ mesalock_sgx = ["sgx_tstd", "mesatee_core/mesalock_sgx"]
 cov = ["sgx_cov"]
 
 [dependencies]
+prost = { version = "0.5.0" }
+bytes = { version = ">= 0.4.12"} # unix app depends on 0.4.12, while sgx lib depends on 0.5.0
```

Comment:
Just a note: the revelent PR on prost is https://github.com/danburkert/prost/pull/246 . Once it's merged, I'll upgrade the forked prost.